using-timescaledb/hypertables.md

```diff
@@ -109,26 +120,31 @@ PostGIS geospatial indexes).  During testing, you might check your
 total chunk sizes via the [chunk relation size](#chunk_relation_size)
 function.
 
-**Space partitions**: The use of additional partitioning is a very
-specialized use case.  **Most users will not need to use it.**
+**Space partitions**: Space partitioning is optional but can make
+sense for certain types of data or when using distributed hypertables.
+**In most cases it is not needed for non-distributed hypertables.**
 
 Space partitions use hashing: Every distinct item is hashed to one of
-N buckets.  Remember that we are already using (flexible) time
-intervals to manage chunk sizes; the main purpose of space
-partitioning is to enable parallel I/O to the same time interval.
+N buckets. In a distributed hypertable, each bucket corresponds to a
+specific data node (although two or more buckets could map to the same
+node). Remember that we are already using (flexible) time intervals to
+manage chunk sizes; the main purpose of space partitioning is to
+enable parallel I/O to the same time interval, both within and across
+nodes.
 
 Parallel I/O can benefit in two scenarios: (a) two or more concurrent
-queries should be able to read from different disks in parallel, or
-(b) a single query should be able to use query parallelization to read
-from multiple disks in parallel.
+queries should be able to read from different nodes and/or disks in
+parallel, or (b) a single query should be able to use query
+parallelization to read from multiple nodes and/or disks in parallel.
 
 Note that query parallelization in PostgreSQL 9.6 (and 10) does not
-support querying *different* hypertable chunks in parallel;
-query parallelization only works on a single physical table (and thus
-a single chunk). We might add our own support for this, but it is not
-currently supported.
+support querying *different* hypertable chunks in parallel; query
+parallelization only works on a single physical table (and thus a
+single chunk). We might add our own support for this, but it is not
+currently supported. Distributed hypertables only work with PostgreSQL
+11.
```

Comment:
Suggest to remove the sentence that we might add it later. This will only pain us.